elasticsearch/src/main/java/org/hibernate/search/elasticsearch/impl/DefaultElasticsearchService.java

```diff
@@ -48,6 +50,10 @@
 	 */
 	private static final String CLIENT_SCOPE_NAME = "default";
 
+	private static final String QUERY_PROPERTIES_PREFIX = "hibernate.search.";
```

Comment:
There's no need to use this prefix, we automatically pre-mask all read properties with an appropriate prefix.

More importantly, we should never force this specific prefix usage as other integrators use a custom prefix for the Search component.

But since I see this was an existing problem I'll take the PR and send a follow-up fix.